mailserver/mailserver.go

```diff
@@ -243,9 +266,9 @@ func (s *WMailServer) processRequest(peer *whisper.Peer, lower, upper uint32, bl
 
 	for i.Next() {
 		var envelope whisper.Envelope
-		err = rlp.DecodeBytes(i.Value(), &envelope)
-		if err != nil {
-			log.Error(fmt.Sprintf("RLP decoding failed: %s", err))
+		decodeErr := rlp.DecodeBytes(i.Value(), &envelope)
+		if decodeErr != nil {
+			log.Error(fmt.Sprintf("RLP decoding failed: %s", decodeErr))
```

Comment:
Should we still proceed in such a case or should we put `continue` to process the next envelope?

Reply:
@adambabik I thought the same but I see the original implementation was not skipping.
I think it's better to add a `continue` like you said